templates/windows/chocolateyInstall.ps1.erb

```diff
@@ -0,0 +1,5 @@
+<%- dirnames = get_directories.map {|d| d.path } -%>
+<%- dirnames.each do |dir| -%>
+  <%- windows_path = platform.convert_to_windows_path(dir) -%>
+Copy-Item <%= "#{windows_path.sub("C:", "C:/ProgramData/chocolatey/lib/#{project.name}")}" %> <%= windows_path %> -recurse
```

Comment:
It'd be nice to avoid having them zipped, to avoid running unzip twice. Maybe that doesn't save us anything however, since we're already having to copy them.